cilium-dbg/cmd/force_destroy.go

```diff
@@ -25,18 +25,18 @@ import (
 	"github.com/cilium/cilium/pkg/socketlb"
 )
 
-// cleanupCmd represents the cleanup command
-var cleanupCmd = &cobra.Command{
-	Use:   "cleanup",
+// forceDestroyCmd represents the force-destroy command
+var forceDestroyCmd = &cobra.Command{
+	Use:   "force-destroy",
 	Short: "Remove system state installed by Cilium at runtime",
 	Long: `Clean up CNI configurations, CNI binaries, attached BPF programs,
 bpffs, tc filters, routes, links and named network namespaces.
 
 Running this command might be necessary to get the worker node back into
 working condition after uninstalling the Cilium agent.`,
 	Run: func(cmd *cobra.Command, args []string) {
-		common.RequireRootPrivilege("cleanup")
-		runCleanup()
+		common.RequireRootPrivilege("force-destroy")
```

Comment:
Sorry, totally missed responding here.

So, when I first discovered this sub-command, I did try and use it on a node with agent running, thinking that this would help force reconcile state on the node to possibly remediate something bad on the node. And I did promptly get `Agent should not be running when cleaning up` error.

But there's definitely value in having this command, so how about we rename this to `post-uninstall-cleanup` instead ?

Reply:
`post-uninstall-cleanup` sounds good to me. I think that this naming provides clear enough expectations around when it is safe to use that this would address the reported issue. Users _could_ still decide to use this in other cases at their own risk, but that would be entirely on them to properly understand the implications.